charts/dependency-track/values.yaml

```diff
@@ -118,7 +129,110 @@ frontend:
   apiBaseUrl: ""
   initContainers: []
 
+tls:
+  # -- Enable TLS transport
+  enabled: false
+  # -- Auto-generate self-signed certificates
+  autoGenerated: true
+  # -- Name of a secret containing the certificates
+  existingSecret: ""
+  # -- Path of the certificate file when mounted as a secret
+  certFilename: tls.crt
+  # -- Path of the certificate key file when mounted as a secret
+  certKeyFilename: tls.key
+  # -- Path of the certificate CA file when mounted as a secret
+  certCAFilename: ca.crt
+  # -- Content of the certificate to be added to the secret
+  cert: ""
+  # -- Content of the certificate key to be added to the secret
+  key: ""
+  # -- Content of the certificate CA to be added to the secret
+  ca: ""
+
 ingress:
+  # -- Set to true to enable ingress record generation
   enabled: false
+
+  # -- Create a TLS secret for this ingress record using self-signed certificates generated by Helm
+  selfSigned: false
+
+  # -- Ingress path type
+  pathType: ImplementationSpecific
+
+  # -- Default host for the ingress resource
+  hostname: dependency-track.local.gd
```

Comment:
Thank you for suggesting the use of `example.com`. I appreciate your attention to using recognizable example domains, and I understand the appeal. We chose `dependency-track.local.gd` for the Helm Chart to streamline the setup process for users testing the chart in local development environments like Docker Desktop, Rancher Desktop, k3d, or microk8s. Our goal was to ensure that the chart could be deployed with minimal configuration. `dependency-track.local.gd` resolves directly to `localhost`, which allows for immediate and hassle-free testing.

While `example.com` is a standard choice, it typically resolves to `NXDOMAIN`. This would require users to edit the `values.yaml` file, potentially complicating the initial setup process.

I value your focus on clarity and adherence to conventions. Could we explore other alternatives that address your concerns while still maintaining ease of setup? Your insights are incredibly valuable for refining our approach and ensuring the chart remains user-friendly. I look forward to your thoughts or any further suggestions you might have!